crates/krun/Cargo.toml

```
@@ -1,24 +1,43 @@
 [package]
 name = "krun"
-version = { workspace = true }
-authors = { workspace = true }
-edition = { workspace = true }
-rust-version = { workspace = true }
-description = { workspace = true }
-repository = { workspace = true }
-license = { workspace = true }
+version = "0.1.0"
+authors = ["Sergio Lopez <slp@redhat.com>", "Teoh Han Hui <teohhanhui@gmail.com>"]
+edition = "2021"
+rust-version = "1.77.0"
+description = "Run programs from your system in a microVM"
+repository = "https://github.com/slp/krun"
+license = "MIT"
 
 [dependencies]
-anyhow = { workspace = true, features = ["std"] }
-bpaf = { workspace = true, features = [] }
-env_logger = { workspace = true, features = ["auto-color", "humantime", "unstable-kv"] }
-krun-sys = { workspace = true, features = [] }
-log = { workspace = true, features = ["kv"] }
-nix = { workspace = true, features = ["user"] }
-rustix = { workspace = true, features = ["process", "std", "use-libc-auxv"] }
-serde = { workspace = true, features = [] }
-serde_json = { workspace = true, features = ["std"] }
-utils = { workspace = true, features = [] }
+anyhow = { version = "1.0.82", default-features = false, features = ["std"] }
+bpaf = { version = "0.9.11", default-features = false }
+byteorder = { version = "1.5.0", default-features = false, features = ["std"] }
+env_logger = { version = "0.11.3", default-features = false, features = ["auto-color", "humantime", "unstable-kv"] }
+#krun-sys = { version = "1.9.1", default-features = false }
+krun-sys = { path = "../krun-sys", default-features = false }
```

This kind of use case is explicitly supported: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#local-paths-in-published-crates

```suggestion
krun-sys = { path = "../krun-sys", version = "1.9.1", default-features = false, features = [] }
```

---

Nice, let's do this, thanks!